spec/feature/user_manage_groups_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.feature 'Managing groups' do
+  scenario 'users can create a group' do
+    feature_login
+    group = build(:group)
+
+    click_link 'Create Group'
```

Comment:
The `Create Group` link is inside a hidden `div`, but it's still present on the HTML page which is why Capybara can locate it anyway.